onward/app/weather/controllers/LocationsController.scala

```diff
@@ -34,44 +34,27 @@ class LocationsController(weatherApi: WeatherApi, val controllerComponents: Cont
       def getEncodedHeader(key: String) =
         request.headers.get(key).map(java.net.URLDecoder.decode(_, "latin1"))
 
+      val maybeCountry = getEncodedHeader(CountryHeader).filter(_.nonEmpty)
       val maybeCity = getEncodedHeader(CityHeader).filter(_.nonEmpty)
       val maybeRegion = getEncodedHeader(RegionHeader).filter(_.nonEmpty)
-      val maybeCountry = getEncodedHeader(CountryHeader).filter(_.nonEmpty)
-
-      (maybeCity, maybeRegion, maybeCountry) match {
-        case (Some(city), Some(region), Some(country)) =>
-          CitiesLookUp.getLatitudeLongitude(CityRef.makeFixedCase(city, region, country)) match {
-            case Some(latitudeLongitude) =>
-              log.info(s"Matched $city, $region, $country to $latitudeLongitude")
 
-              weatherApi.getNearestCity(latitudeLongitude) map { location =>
```

Comment:
@jamesgorrie @mxdvl do you remember why removed the search by latitute / longitude?

cc @abeddow91 @cemms1